### ISSUE TITLE
Integration with locator bar in QGIS 3

### ISSUE THREAD

**Issue by Gustry** [xcaeag/Nominatim-Qgis-Plugin#4]
Hi,
Thanks for the plugin.

Would it be possible to integrate the locator bar in QGIS 3?
It would be very handy to quickly to open the locator bar, having a small prefix and quickly zoom to a nominatim result instead of using its own plugin panel.

Thanks

**Comment by kannes** [comment]
I agree that this would be a nice extension of this plugin! There is an alternative that does _just_ that: https://github.com/rduivenvoorde/nominatim_locator_filter

**Comment by Gustry** [comment]
Indeed, I discovered it after the creation of this ticket. Thanks for the reminder.

**Comment by xcaeag** [comment]
Hello,

it's a good idea of course, but it is because there was already such a plugin that nothing was done in this direction.
I didn't want to be in competition either !

The cohabitation of the two is not inconvenient, and what will it bring better ?
